### ISSUE TITLE
API Tracking issue: profile.groups.read

### ISSUE THREAD

**Issue by bibliotechie** [bibliotechie/antithesis#25]
This is a tracking issue for the Hypothesis API endpoint `GET /profile/groups`. Corresponding wiki page: https://github.com/bibliotechie/antithesis/wiki/Hypothesis-API%3A-profile.groups.read

**Comment by bibliotechie** [comment]
I'm breaking out portions needed for MVP into separate issue, in case we implement the full feature later.

**Comment by bibliotechie** [comment]
Depends on #34.
